package.json

```
@@ -8,7 +8,9 @@
   },
   "scripts": {
     "flow": "flow",
-    "test": "flow check && mocha tests"
+    "test": "flow check && npm run prettier:check && mocha tests",
+    "prettier:check": "prettier 'lib/**/*.js' --list-different",
```

Looks like the single quotes are the problem: https://github.com/prettier/prettier/issues/4561#issuecomment-392817015

Try it without single quotes:
```suggestion
    "prettier:check": "prettier \"lib/**/*.js\" --list-different",
```

If that works, we should give `prettier:write` the same treatment!